### PR TITLE
Refactor wordsmith updates

### PR DESCRIPTION
**Pull request by elisabethnewell**

### CODE REVIEW

File: content/references.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 title: 'Reference'
+metaTitle: 'Reference'
+metaDescription: 'Links to helpful reference articles for building on Element.'
 ---
 
-- [Visual Guide to Proptypes](references/proptypes)
+- [Proptypes](references/proptypes)
 - [Block Utils](references/block-utils)
-- [Volusion API ElementSdk Client Reference](references/sdk)
+- [SDK](references/sdk)
```
Comment (chris-decker-volusion) [inline content/references.md]: @elisabethnewell This section is specifically for `ElementSdk.client`. Do you still want to call it just `SDK`?

Comment (elisabethnewell) [reply content/references.md]: The page title is just "SDK," do we want to change that to "Element SDK"?

Comment (chris-decker-volusion) [reply content/references.md]: As long as we're not changing the file name, we can change the title. But I'm saying it's not just the SDK. It's specifically the client content within the SDK.

Comment (elisabethnewell) [reply content/references.md]: Would "Element SDK Client" suffice?

Comment (chris-decker-volusion) [reply content/references.md]: Sure.

**Comment by chris-decker-volusion** [comment]
Deploying.

**Pull request merged**